CHANGELOG.md

```diff
@@ -5,13 +5,32 @@
 ### State Machine Breaking
 
 - [cronos#241](https://github.com/crypto-org-chain/cronos/pull/241) Update ethermint to main and merged statedb refactoring in custom fork.
+- [cronos#289](https://github.com/crypto-org-chain/cronos/pull/289) Update ethermint to `v0.10.0`, with ibc-go reverted to `v2.0.2`, update cosmos-sdk to `v0.45.0-rc1`, the cronos related changes in the dependencies are:
```

Comment:
`the cronos related changes in the dependencies are:` 

Is it missing something in the sentence? or you it refers to all what is below?

Reply:
maybe just `Update ethermint to `v0.10.0-cronos` which uses ibc-go `v2.0.2` instead of `v3.0.0-alpha2`

Reply:
updated, thanks for suggestion.